models/Document/Page.php

```diff
@@ -102,13 +102,6 @@ public function setTitle(string $title): static
         return $this;
     }
 
-    public function setMetaData(array $metaData): static
-    {
-        $this->metaData = $metaData;
-
-        return $this;
-    }
-
     public function getMetaData(): array
```

Comment:
The getter and the property `metaData` needs to be removed as well. 
Also we need to remove the `metaData` column from the `documents_page` table in `install.sql` and we need a migration for that.